at_response.c

```diff
@@ -478,8 +478,9 @@ static int at_response_error (struct pvt* pvt, at_res_t res)
 				break;
 
 			case CMD_AT_CMGR:
-				ast_log (LOG_ERROR, "[%s] Error reading SMS message\n", PVT_ID(pvt));
-				at_retrieve_next_sms(&pvt->sys_chan);
+				if (!IS_SUPPRESS_ERROR_ENABLED(ecmd->flags))
+					ast_log (LOG_ERROR, "[%s] Error reading SMS message\n", PVT_ID(pvt));
```

Comment:
Please use 0 spaces between function and parenthesis (`ast_log(`).

Reply:
Additionally, I think we'll want a ast_debug() if it is suppressed. If people are wondering what the dongle is doing that might come in handy.

Reply:
ATQ_CMD_FLAG_SUPPRESS_ERROR suggests that it's generic, but it looks like it only works for sms.

Might it be better to replace ast_log() with `log_error(enum suppress, ...)` and use that everywhere in this function?

Reply:
ok, will fix it soon